internal/mysqld/server.go

```diff
@@ -0,0 +1,50 @@
+package mysqld
+
+import (
+	"fmt"
+
+	"github.com/milvus-io/milvus/internal/types"
+	"github.com/xelabs/go-mysqlstack/driver"
+	"github.com/xelabs/go-mysqlstack/xlog"
+	"go.uber.org/zap/zapcore"
+)
+
+type Server struct {
+	listener *driver.Listener
+}
+
+func (s *Server) Start() error {
```

Comment:
It is recommended to add waitgroup and wait for the background coroutine in Start to exit when Close